src/main/java/frc/robot/subsystems/intakerelease/IntakeRelease.java

```diff
@@ -54,10 +53,10 @@ public void periodic() {
 
     switch (state) {
       case DISABLED -> intakeReleaseIO.setPercent(0);
-      case INTAKE -> intakeReleaseIO.setPercent(-0.5);
-      case RELEASE -> intakeReleaseIO.setPercent(0.2);
-      case HOLD_CUBE -> intakeReleaseIO.setPercent(-0.03);
-      case HOLD_CONE -> intakeReleaseIO.setPercent(-0.1);
+      case INTAKE -> intakeReleaseIO.setPercent(-0.7);
```

Comment:
Have these new values been tested?  I know we were seeing some issues with holding - will these resolve them?